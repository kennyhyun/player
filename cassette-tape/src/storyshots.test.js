const { default: initStoryshots, multiSnapshotWithOptions } = require('@storybook/addon-storyshots');
const { render } = require('enzyme');

beforeEach(() => {
  Date.now = jest.fn(() => 1541586333599);
});

initStoryshots();
/*
{
  renderer: render,
  test: multiSnapshotWithOptions({
    // https://github.com/storybooks/storybook/tree/master/addons/storyshots#using-createnodemock-to-mock-refs
    // https://reactjs.org/blog/2016/11/16/react-v15.4.0.html#mocking-refs-for-snapshot-testing
    createNodeMock: element => {
      if (element.type === 'div') {
        return document.createElement('div');
      }

      if (element.type === 'input') {
        return document.createElement('input');
      }

      if (element.type === 'textarea') {
        return document.createElement('textarea');
      }
    },
  }),
});
*/
