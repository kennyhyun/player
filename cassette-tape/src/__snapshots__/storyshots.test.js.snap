// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Cassete tapes Cassette 1`] = `
<div>
  <div
    onClick={[Function]}
    pos={0}
    sideindex={0}
    style={
      Object {
        "backgroundImage": "url(/Cassette.svg)",
        "height": "400px",
        "margin": "auto",
        "position": "relative",
        "width": "600px",
      }
    }
  >
    <div
      className="Reel-reelTape-0-1-1 Reel-reelTape-0-1-3"
      style={
        Object {
          "height": "263.8px",
          "transform": "translate3d(0, 0, 0) translateX(-131.9px) translateY(-131.9px)",
          "width": "263.8px",
        }
      }
    />
    <div
      className="Reel-reelCore-0-1-2 Reel-reelCore-0-1-4"
      style={
        Object {
          "height": "138.2px",
          "transform": "translate3d(0, 0, 0) translateX(-69.1px) translateY(-69.1px)",
          "width": "138.2px",
        }
      }
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-7"
      onTransitionEnd={[Function]}
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-7 Wheel-animating-0-1-6"
      onTransitionEnd={[Function]}
      style={
        Object {
          "--wheel-radius": "-89.9px",
        }
      }
    />
    <div
      className="Reel-reelTape-0-1-1 Reel-reelTape-0-1-8"
      style={
        Object {
          "height": "139.2px",
          "transform": "translate3d(0, 0, 0) translateX(-69.6px) translateY(-69.6px)",
          "width": "139.2px",
        }
      }
    />
    <div
      className="Reel-reelCore-0-1-2 Reel-reelCore-0-1-9"
      style={
        Object {
          "height": "138.2px",
          "transform": "translate3d(0, 0, 0) translateX(-69.1px) translateY(-69.1px)",
          "width": "138.2px",
        }
      }
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-10"
      onTransitionEnd={[Function]}
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-10 Wheel-animating-0-1-6"
      onTransitionEnd={[Function]}
      style={
        Object {
          "--wheel-radius": "-89.9px",
        }
      }
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-11"
      onTransitionEnd={[Function]}
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-11 Wheel-animating-0-1-6"
      onTransitionEnd={[Function]}
      style={
        Object {
          "--wheel-radius": "-15px",
        }
      }
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-12"
      onTransitionEnd={[Function]}
    />
    <div
      className="Wheel-wheel-0-1-5 Wheel-wheel-0-1-12 Wheel-animating-0-1-6"
      onTransitionEnd={[Function]}
      style={
        Object {
          "--wheel-radius": "-15px",
        }
      }
    />
  </div>
</div>
`;
